spec/unit/query_components/aggregates_spec.rb

```diff
@@ -1,166 +0,0 @@
-require 'spec_helper'
-
-RSpec.describe QueryComponents::Aggregates do
-  def make_search_params(aggregates:, filters: [])
-    Search::QueryParameters.new(
-      filters: filters, aggregates: aggregates, debug: { include_withdrawn: true }
-    )
-  end
-
-  context "search with aggregate" do
-    it "have correct aggregate in payload" do
-      builder = described_class.new(
-        make_search_params(
-          aggregates: { "organisations" => { requested: 10, scope: :exclude_field_filter } },
-        )
-      )
-
-      result = builder.payload
-
-      expect(
-        "organisations" => {
-          filter: { match_all: {} },
-          aggs: {
-            "filtered_aggregations" => {
-              terms: {
-                field: "organisations",
-                order: { _count: "desc" },
-                size: 100000,
-              }
-            }
-          }
-        },
-        "organisations_with_missing_value" => {
-          filter: { match_all: {} },
-          aggs: {
-            "filtered_aggregations" => {
-              missing: { field: "organisations" }
-            }
-          }
-        },
-      ).to eq(result)
-    end
-  end
-
-  context "search with aggregate and filter on same field" do
```

Comment:
Can I confirm we have an integration test for this feature?

Reply:
https://github.com/alphagov/rummager/blob/de6d015ea376f3b410c842ce284d18360efddfea/spec/integration/search/search_spec.rb#L202

Reply:
😄